### PR TITLE
[RTD]: Fixing left side menu

### LINKED ISSUE #1889: [TSC RTD] Make the left side menu more clear
Left side menu in readthedocs needs the following changes: 
- Less redundant CNTT mentions
- Show the two options of RA-s, RI-s and RC-s

### PR DESCRIPTION
**Pull request by CsatariGergely**
closes #1889

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

File: doc/ref_cert/README.md [hunk 1]
```diff
@@ -18,24 +20,24 @@ If there is no clear traceability and strong links between these 3 components, t
 * Reduce ambiguity in testing, requirements, and conformance
 
 
-## Requirement Principles
+### Requirement Principles
 Requirements Principles can be found in the [RM Principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#2.0)
```
Comment (tomkivlin) [inline doc/ref_cert/README.md:24]:
```suggestion
Requirement Principles can be found in the [RM Principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#2.0)
```

**Review by pgoyal01 — APPROVED** [review]
Approve subject to correction suggested by @tomkivlin

File: doc/ref_cert/README.md [hunk 2]
```diff
@@ -18,24 +20,24 @@ If there is no clear traceability and strong links between these 3 components, t
...
 
-## Testing Principles
+### Testing Principles
 * There must be traceability between test cases and requirement being validated
 * Failures should provide additional content to inform the user where or how the requirement was violated (ex: which file or resource violated the requirement).  Put another way, don’t require the user to read the test to understand what went wrong
```
Comment (tomkivlin) [inline doc/ref_cert/README.md:28]: Does `ex:` mean `example` or `excluding`.  Suggest either using `e.g.` or `excluding`.

**Review by ulikleber — APPROVED** [review]

**Review by gkunz — APPROVED** [review]

File: doc/ref_cert/README.md (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_cert/README.md:28]: I've created a separate pr to fix this in #2197

**Review by tomkivlin — APPROVED** [review]

**Pull request merged**